Chessington.GameEngine/Helpers.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Chessington.GameEngine
+{
+	public class Helpers
+    {
+        public List<Direction> AddMovesAndUpdateDirections(Board board, List<Direction> directions, Square currentSquare, int i, List<Square> moves, Player player)
+        {
+            var toRemove = new List<Direction>();
+            foreach (var direction in directions)
+            {
+                var potentialMove = Square.At(currentSquare.Row + i * direction.RowOffset,
```

Comment:
When splitting a method invocation over multiple lines it's best practice to put each argument on a new line:
```
var potentialMove = Square.At(
  currentSquare.Row + i * direction.RowOffset,
  currentSquare.Col + i * direction.ColOffset
)
```
🙂